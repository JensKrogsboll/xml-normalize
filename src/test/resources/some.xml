<?xml version="1.0"?>
<catalog>
    <product product_image="cardigan.jpg" description="Cardigan Sweater">
        <catalog_item gender="Men's" age="2">
            <item_number>RRX9856</item_number>
            <price>39.95</price>
            <size description="Medium">
                <color_swatch image="red_cardigan.jpg">Red</color_swatch>
                <color_swatch image="burgundy_cardigan.jpg">Burgundy</color_swatch>
            </size>
            <size description="Large">
                <color_swatch image="burgundy_cardigan.jpg">Burgundy</color_swatch>
                <color_swatch image="red_cardigan.jpg">Red</color_swatch>
            </size>
        </catalog_item>
        <catalog_item age="5" gender="Women's">
            <item_number>QWZ5671</item_number>
            <price>42.50</price>
            <size description="Small">
                <color_swatch image="red_cardigan.jpg">Red</color_swatch>
                <color_swatch image="burgundy_cardigan.jpg">Burgundy</color_swatch>
                <color_swatch image="navy_cardigan.jpg">Navy</color_swatch>
            </size>
            <size description="Medium">
                <color_swatch image="burgundy_cardigan.jpg">Burgundy</color_swatch>
                <color_swatch image="red_cardigan.jpg">Red</color_swatch>
                <color_swatch image="navy_cardigan.jpg">Navy</color_swatch>
                <color_swatch image="black_cardigan.jpg">Black</color_swatch>
            </size>
            <size description="Large">
                <color_swatch image="navy_cardigan.jpg">Navy</color_swatch>
                <color_swatch image="black_cardigan.jpg">Black</color_swatch>
            </size>
            <size description="Extra Large">
                <color_swatch image="black_cardigan.jpg">Black</color_swatch>
                <color_swatch image="burgundy_cardigan.jpg">Burgundy</color_swatch>
            </size>
        </catalog_item>
    </product>
</catalog>