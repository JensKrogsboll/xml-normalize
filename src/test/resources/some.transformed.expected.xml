<?xml version="1.0" encoding="UTF-8"?><catalog>
  <product description="Cardigan Sweater" product_image="cardigan.jpg">
    <catalog_item age="5" gender="Women's">
      <size description="Extra Large">
        <color_swatch image="black_cardigan.jpg">Black</color_swatch>
        <color_swatch image="burgundy_cardigan.jpg">Burgundy</color_swatch>
      </size>
      <size description="Medium">
        <color_swatch image="black_cardigan.jpg">Black</color_swatch>
        <color_swatch image="burgundy_cardigan.jpg">Burgundy</color_swatch>
        <color_swatch image="navy_cardigan.jpg">Navy</color_swatch>
        <color_swatch image="red_cardigan.jpg">Red</color_swatch>
      </size>
      <size description="Large">
        <color_swatch image="black_cardigan.jpg">Black</color_swatch>
        <color_swatch image="navy_cardigan.jpg">Navy</color_swatch>
      </size>
      <size description="Small">
        <color_swatch image="burgundy_cardigan.jpg">Burgundy</color_swatch>
        <color_swatch image="navy_cardigan.jpg">Navy</color_swatch>
        <color_swatch image="red_cardigan.jpg">Red</color_swatch>
      </size>
    </catalog_item>
    <catalog_item age="2" gender="Men's">
      <size description="Medium">
        <color_swatch image="burgundy_cardigan.jpg">Burgundy</color_swatch>
        <color_swatch image="red_cardigan.jpg">Red</color_swatch>
      </size>
      <size description="Large">
        <color_swatch image="burgundy_cardigan.jpg">Burgundy</color_swatch>
        <color_swatch image="red_cardigan.jpg">Red</color_swatch>
      </size>
    </catalog_item>
  </product>
</catalog>
